### PR TITLE
update generate docs

### PR DESCRIPTION
**Pull request by JimBugwadia**
Fixes: https://github.com/kyverno/website/issues/28

### CODE REVIEW

File: content/en/docs/Writing policies/generate.md [hunk 1]
```diff
@@ -4,14 +4,26 @@ description: Create additional resources based on resource creation, or label/me
...
+## Updating existing resources
 
+Cenerate rules are triggered during API CREATE operations, or API UPDATE operations that satisfy the match condition of a generate rule. By default, generate requests are not applied to existing resources. This prevents inadvertent changes to existing resources.
 
-## Generate a ConfigMap using inline data
+To trigger a generate request for existing resources, you can write the generate rule to match based on a label or annotation, and then apply the label or annotation to resources you want to have the generate apply to. 
+
+For example, you can write a generate rule that adds a network policy to all namespaces with the annotation `kyverno.io/netpol: true` and a mutate rule to add that annotation to all new namespaces. With these rules in place, you can then modify any other namespace to add the annotation, and Kyverno will then execute the generate rule for that namespace.
+
```
Comment (realshuting) [inline content/en/docs/Writing policies/generate.md:18]: Let's add an example:
````yaml
apiVersion: kyverno.io/v1
kind: ClusterPolicy
metadata:
  name: add-serviceaccount
spec:
  background: false
  rules:
  - name: add-ns-label
    match:
      resources:
        kinds:
        - Namespace
    mutate:
      patchStrategicMerge:
        metadata:
          labels:
            kyverno.io/netpol: "true"
  - match:
      resources:
        kinds:
        - Namespace
        selector:
          matchLabels:
            kyverno.io/netpol: "true"
    name: default-deny-ingress
    validationFailureAction: audit
    generate:
      kind: NetworkPolicy
      name: default-deny-ingress
      namespace: "{{request.object.metadata.name}}"
      synchronize: true
      data:
        spec:
          # select all pods in the namespace
          podSelector: {}
          policyTypes:
          - Ingress
````

**Comment by chipzoller** [comment]
I've basically covered this now.

**Comment by chipzoller** [comment]
See if this is satisfactory: https://github.com/kyverno/website/commit/10fb2972725c016a7d1367a23f3d6a7aaa7bd24f

File: content/en/docs/Writing policies/generate.md (hunk 1 above)
Comment (JimBugwadia) [reply content/en/docs/Writing policies/generate.md:18]: resolved by https://github.com/kyverno/website/commit/10fb2972725c016a7d1367a23f3d6a7aaa7bd24f

**Review by chipzoller — APPROVED** [review]

**Review by evalsocket — APPROVED** [review]

**Pull request merged**